od.proto

```diff
@@ -40,15 +40,39 @@ message OD {
 // TODO Do we want to model multi-modal trips, like driving to a park-and-ride
 // and taking transit? Or using an e-scooter for last mile?
 enum Mode {
-  // Single-occupancy vehicle? Rideshare? Passengers vs drivers?
-  Driving = 0;
-  Cycling = 1;
-  Walking = 2;
-  // Distinguish buses; ferries, trains?
-  PublicTransit = 3;
-  Carpool = 4;
-  ElectricMicromobility = 5;
-  // More?
+  // Human powered modes
+  WALK = 0;
+  JOG = 1;
+  PUSH_SCOOTER = 2;
+  PEDAL_CYCLE = 3;
+
+  // Electric mobility and 2 wheelers
+  MICROMOBILITY = 4;
+  ELECTRIC_SCOOTER = 5;
+  ELECTRIC_CYCLE = 6;
+  MOBILITY_SCOOTER = 7;
+  MOTORCYCLE = 8;
+  // From STATS19
+  // Motorcycle - unknown cc 125cc and under 50cc
+  // over 125cc up to 500cc Motorcycle over 500cc
+
+  // Public transport
+  PUBLIC_TRANSPORT = 9;
```

Comment:
What about `PUBLIC_TRANSIT_OTHER`? Copying from Slack: "It's weird to mix a general category with specific things. If someone is looking at all the enum values, they intuitively would want to sum them all up and not worry about double-counting. All of the enum values should be mutually exclusive"

Reply:
Sounds reasonable but what about cases where you don't know? I'm thinking of data collection sources where you only have 'public transport'. I'm trying to think of scenarios where this could lead to double counting. To me these options are about ensuring people have the right 'bins' into which they can put their diverse transport data. If someone chooses to to count buses and public transport they should do so knowing what's going on. Similar with `CAR` categories.

Reply:
When you don't know, use `PUBLIC_TRANSPORT_OTHER`.

Another way of phrasing this: As somebody trying to convert my OD data into this schema, I should be able to unambiguously choose a `Mode` enum value. `BUS` is a refinement of `PUBLIC_TRANSPORT`, so I'm confused. `PUBLIC_TRANSPORT_OTHER` or `PUBLIC_TRANSPORT_UNKNOWN` are both distinct from `BUS`; there's no overlap between these sets.